### PR TITLE
Modulize and some html fixes

### PR DESCRIPTION
**Pull request by Claymanus**

### CODE REVIEW

File: site/js/deregistration.js [hunk 1]
```diff
@@ -3,33 +3,31 @@ import { _ } from './modules/gettext'
 import { gettext } from './modules/gettext'
 import PageScript from './modules/script.js'
 import { setup_the_deregister_form_buttons } from './modules/setup_buttons'
-
+import * as Ajax from './modules/ajax'
```
Comment (magwas) [inline site/js/deregistration.js]: Importing * makes dependencies less explicit.

Comment (Claymanus) [inline site/js/deregistration.js]: Nem vágom a problémád

Comment (Claymanus) [inline site/js/deregistration.js]: Igy minden fv elérhető a modulból adott esetben az Ajax.fv néven. És csak azok a fv-k rollupolódnak a deploy kódba, amiket használunk is, nem az egész hóbelebanc

File: site/js/deregistration.js [hunk 2]
```diff
@@ -88,14 +57,14 @@ PageScript.prototype.doDeregister = function() {
 				csrf_token: $this.getCookie("csrf"),
 				deregister_secret: $this.QueryString.secret
 			}
-			$this.ajaxpost( "/v1/deregister_doit", post, $this.callback( deregisterCallback ) )
+			Ajax.post( "/v1/deregister_doit", post, { next: deregisterCallback } )
```
Comment (magwas) [inline site/js/deregistration.js]: I like this change. Good direction.

File: site/js/modules/ajax.js [hunk 3]
```diff
@@ -41,72 +56,67 @@ var Ajax = function( args ){
...
-    $this.valaidateCallbackArgs=function(args){
-        var next=null,
-            error=null;
-        if (typeof args != "undefined") {
-            next = args.next || null;
-            error = args.error || null
-        }
-        return {next: next, error: error }
-    }
+export function valaidateCallbackArgs( args ){
```
Comment (magwas) [inline site/js/modules/ajax.js]: s/valaidate/validate/

File: site/js/modules/pagehandler.js [hunk 4]
```diff
@@ -0,0 +1 @@
+//page handler
```
Comment (magwas) [inline site/js/modules/pagehandler.js]: If a module is empty, we do not need it.

**Comment by magwas** [comment]
Please note that the merge request is accepted with comments.
Further comments:
- I would advocate to use the one class/one module approach. A module should contain one class even if it contains only static functions. Makes easier to understand the structure.
- gettext looks like as the only thing imported with lover case. Again, conventions help to understand the structure easier.

**Pull request merged**